zeebe/qa/integration-tests/src/test/java/io/camunda/zeebe/it/client/command/ThrowErrorTest.java

```diff
@@ -62,9 +63,12 @@ public void shouldThrowError(final boolean useRest, final TestInfo testInfo) {
     final long jobKey = activateJob(client, useRest, jobType).getKey();
 
     // when
-    getCommand(client, useRest, jobKey).errorCode(ERROR_CODE).send().join();
+    final ThrowErrorResponse response =
+        getCommand(client, useRest, jobKey).errorCode(ERROR_CODE).send().join();
 
     // then
+    assertThat(response).isNotNull().isInstanceOf(ThrowErrorResponse.class);
```

Comment:
❓ what is the reasoning for this assertion?

Reply:
The reasoning is to check that response is actually not `null` (like it was beforehand, actually for both gRPC and REST in this one case 😅).